eloqua/eloqua-connector/eloqua-connector-1.0.0/org.wso2.carbon.connector/src/main/resources/customObjects/createCustomObjectImport.xml

```diff
@@ -113,6 +117,9 @@
 			if (uri != null && uri != "") {
                payload["uri"]=uri;
             }
+            if (isSyncTriggeredOnImport != null && isSyncTriggeredOnImport != "") {
+               payload["isSyncTriggeredOnImport"]=isSyncTriggeredOnImport;
```

Comment:
Seems not formatted correctly. Leave a space before and after the equal sign (=).